openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
+    """Enforces correct order of features.
+
+    Alphabetically orders the feature columns. The load column remains the first column
+        and the Horizons column remains the last column. Everything in between is alphabetically sorted.
+
+    Args:
+        df (pd.DataFrame): Dataframe with features.
+
+    Returns:
+        pd.DataFrame with features with all columns in the correct order
+    """
+
+    # Extract first column name
+    first_col_name = df.columns.to_list()[0]  # Most of the time this is load
+
+    # Sort columns
+    columns = list(np.sort(df.columns.to_list()))
+
+    # Remove first column and add to the start
+    columns.remove(first_col_name)
+    col_order = [first_col_name] + columns
```

Comment:
See above, if you choose `col` as an abbreviation for `columns` this is fine. But if you go for the more descriptive naming this should be `column_order`. 

I guess `sorted_columns` would make more sense here btw.